### ISSUE TITLE
[3.1] When you batch add taxa AND vouchers, vouchers do not get linked to the checklists

### ISSUE THREAD

**Issue by themerekat** [Symbiota/Symbiota#1828]
In /checklists/voucheradmin.php
using the MissingTaxa tab, when you batch link taxa & vouchers
![image](https://github.com/user-attachments/assets/10267e79-0e83-4d70-b3d3-e7fcb4f788e6)
and you leave the box unchecked
![image](https://github.com/user-attachments/assets/bb5c7142-1b56-4a24-a1cc-aa649d25430c)
The name gets added, but not the vouchers
![image](https://github.com/user-attachments/assets/6fdcbcb3-9d15-4069-9862-3336c2de8d6e)
(yes Notes & Vouchers is supposed to be on in this view)

**Comment by egbot** [comment]
Issue has been resolved within the next hotfix (v3.1.4). Will close this issue once hotfix is merged into Master branch.
